index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<section class="container">
+    <header class="Header">
+        <a class="mylogo">MY logo</a>
+         <nav>
+            <img src="./image/logobuger.JPG">
+         </nav>
+    </header>
+<section class="main-section">
+            <h1 class="main-header"> I'm Mihertab Glad to see you!</h1>
+            <p class = "home-text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] Kindly make this paragraph's `font-family` Poppins to match the design in Figma 👍